src/redux/action-creators/readArticle.js

```diff
@@ -9,11 +9,20 @@ import {
 
 const readArticle = slug => (dispatch) => {
   const url = `${process.env.REACT_APP_API}/articles/${slug}`;
+  const ratingUrl = `${process.env.REACT_APP_API}/articles/${slug}/ratings`;
   dispatch({ type: FETCH_ARTICLE_START, payload: {} });
   return axios
     .get(url, optRequest)
-    .then((response) => {
+    .then(async (response) => {
```

Comment:
I think here you can stick with then-catch promise or async-await instead of chaining them  for code consistency

Reply:
resolved